### PR TITLE
New | Calendly | Created a calendar section using calendly

### PR DESCRIPTION
**Pull request by mhindawash**
### **Description:**

Added a calendar section using calendly to allow for appointments as well as zoom appointments to do remote design. Updated the github pullrequest template to include a screenshots section as with this project it's mostly design and there needs to be some evidence to compare for anyone reviewing the project. Also removed the sanity_cli as I am no longer going to go that direction. Changed some of the naming conventions per request for some of the sections.

### **Issue Linked**

N/A

### **Screenshots**

image
image
image

### CODE REVIEW

File: components/Gallery.tsx
Comment (clarkg) [inline components/Gallery.tsx:1]: all the changes here don't seem related to Calendly?  definitely seem big enough to their own PR.  or maybe you didn't intend to include them in your PR?

File: components/CalendlyComponent.tsx
Comment (clarkg) [inline components/CalendlyComponent.tsx:1]: any particular reason you rolled your own component instead of just using this or something else?

https://www.npmjs.com/package/react-calendly

File: components/Process.tsx
Comment (clarkg) [inline components/Process.tsx:1]: is this change intended?  seems not related at the very least

File: styles/BookingPage.module.css
Comment (clarkg) [inline styles/BookingPage.module.css:1]: feels confusing to use this and tailwind at the same time

File: components/CalendlyComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const CalendlyComponent: React.FC = () => {
+	useEffect(() => {
+		const head = document.querySelector("head");
+		const script = document.createElement("script");
+		script.setAttribute(
+			"src",
+			"https://assets.calendly.com/assets/external/widget.js"
+		);
+		head?.appendChild(script);
+	}, []);
```
Comment (clarkg) [inline components/CalendlyComponent.tsx]: this feels suss rather than re-using an existing Calendly component off the shelf

**Review by clarkg — COMMENTED** [review]
Screenshots look great!  

I haven't been using "Request changes" on your website since I don't want to actually block you, but my feedback would be blocking otherwise

File: components/CalendlyComponent.tsx
Comment (mhindawash) [reply components/CalendlyComponent.tsx:1]: I didn't know this was a thing to be honest.

File: components/Gallery.tsx
Comment (mhindawash) [reply components/Gallery.tsx:1]: Yeah it's probably just the changes I brought from my other branches somehow. But I also did do some other changes but the key change was the calendar section being added

File: components/CalendlyComponent.tsx (hunk 1 above)
Comment (mhindawash) [reply components/CalendlyComponent.tsx]: I figured it would look like that but this is how I managed to make it work so I stuck with it. 

"if it aint broke dont fix it" mentality if you will haha.

File: styles/BookingPage.module.css
Comment (mhindawash) [reply styles/BookingPage.module.css:1]: I was having difficulty with the styling on the tailwind part since it's still relatively new to me. This isn't a permanent change, I eventually mean to switch back to tailwind.

File: components/Process.tsx
Comment (mhindawash) [reply components/Process.tsx:1]: The formating seems to be different from mac to windows for some reason. I use the same "prettier" though which is weird.
